.gitattributes

```diff
@@ -1,2 +1,3 @@
-*.bat	text	eol=crlf
-*.cmd	text	eol=crlf
+*.bat		text	eol=crlf
+*.cmd		text	eol=crlf
+/bin/diktat	text	eol=lf		diff=bash
```

Comment:
are you sure that path should be absolute here? maybe `bin/diktat`?

Reply:
This is the same syntax as that of `.gitignore`: `bin/diktat` would mean **any** path containing these path elements, e. g. `bin/diktat`, `foo/bin/diktat`, and `bar/bin/diktat`.

`/bin/diktat`, on the other hand, means that the path is relative to the project root, i. e. `${projectRoot}/bin/diktat`.